caffe2/contrib/aten/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-if(BUILD_ATEN)
-  # Add source generated by Codegen.cmake and pass to parent
+# Add source generated by Codegen.cmake and pass to parent
+if (NOT ANDROID AND NOT IOS)
```

Comment:
The variable naming change here does not seem like an improvement. It seems better to keep `BUILD_ATEN` (not user-toggleable, but an internal cmake variable) and set it to `OFF` when `ANDROID OR IOS`.